polkadot/node/core/candidate-validation/src/tests.rs

```diff
@@ -480,95 +480,22 @@ fn candidate_validation_bad_return_is_invalid() {
 	let candidate_receipt = CandidateReceipt { descriptor, commitments_hash: Hash::zero() };
 
 	let v = executor::block_on(validate_candidate_exhaustive(
-		MockValidateCandidateBackend::with_hardcoded_result(Err(
+        MockValidateCandidateBackend::with_hardcoded_result(Err(
 			ValidationError::InvalidCandidate(WasmInvalidCandidate::HardTimeout),
 		)),
-		validation_data,
-		validation_code,
-		candidate_receipt,
-		Arc::new(pov),
-		ExecutorParams::default(),
-		PvfExecTimeoutKind::Backing,
-		&Default::default(),
+        validation_data,
+        validation_code,
+        candidate_receipt,
+        Arc::new(pov),
```

Comment:
I think some formatting is getting messed up (tabs -> spaces). Make sure you are running `cargo +nightly fmt` with a recent nightly, you can set it as a pre-commit git hook. :)